cum/scrapers/madokami.py

```diff
@@ -82,10 +82,10 @@ def download(self):
             else:
                 total_length = int(total_length)
                 with self.progress_bar(total_length) as bar:
-                    for chunk in r.iter_content(chunk_size=1024):
+                    for chunk in r.iter_content(chunk_size=4096):
                         if chunk:
                             bar.update(len(chunk))
                             f.write(chunk)
-                            f.flush()
+                f.flush()
```

Comment:
The behaviour on madokami downloads is different from batoto and dynastyscans downloads in that the files are closed as the scope is left thanks to the `with` statement. As I did not want to take any chances when it comes to submitting yet another broken PR, I left one explicit flush in there at the end of the download, since it doesn't hurt and this is the last time the buffer is being flushed is the only time it matters anyway.